source/docs/advanced/shipping/mondial-relay.md

```diff
@@ -21,44 +21,51 @@ Then, within your Front-Commerce project, you have to:
 
 - Use the module in your `.front-commerce.js`
   ```diff
-// .front-commerce.js
-module.exports = {
+  // .front-commerce.js
+  module.exports = {
   name: "Front Commerce DEV",
   url: "http://www.front-commerce.test",
   modules: [
-+   "./node_modules/front-commerce/modules/shipping-mondialrelay"
-    "./src"
+  ```
+
+* "./node_modules/front-commerce/modules/shipping-mondialrelay"
+  "./src"
   ],
   serverModules: [
-    { name: "FrontCommerce", path: "server/modules/front-commerce" },
-    { name: "Magento2", path: "server/modules/magento2" },
-+   {
-+     name: "MondialRelay",
-+     path: "shipping-mondialrelay/server/modules/magento2-mondialrelay",
-+   },
+  { name: "FrontCommerce", path: "server/modules/front-commerce" },
+  { name: "Magento2", path: "server/modules/magento2" },
+* {
+*     name: "MondialRelay",
+*     path: "shipping-mondialrelay/server/modules/magento2-mondialrelay",
+* },
   ],
   webModules: [
-    { name: "FrontCommerce", path: "./src/web" },
+  { name: "FrontCommerce", path: "./src/web" },
   ],
-};
-```
+  };
+
+````
 - Import styles of MondialRelay related components by overriding the `_modules.scss`
   ```diff
 // src/web/theme/modules/_modules.scss
 
 +@import "~theme/modules/MondialRelay/MondialRelay";
-```
+````
+
 - Import MondialRelay component in by overriding the `getAdditionalDataComponent` used for Shipping methods
   ```diff
-// src/web/theme/modules/Checkout/ShippingMethod/AdditionalShippingInformation/getAdditionalDataComponent.js
-+import MondialRelay from "theme/modules/MondialRelay";
+  // src/web/theme/modules/Checkout/ShippingMethod/AdditionalShippingInformation/getAdditionalDataComponent.js
+  +import MondialRelay from "theme/modules/MondialRelay";
+  ```
 
 const ComponentMap = {
-+ mondialrelay: {
-+   pickup: MondialRelay,
-+ }
-};
-```
+
+- mondialrelay: {
```

Comment:
Addition markers replaced with minus markers here ?!

Reply:
It was related to the previous diff related error: Prettier considered it as a Markdown list item instead of a diff addition.

Fixed in bc38563675bc4e83fee9c6aaa41060e395997968